provisioning/miniprov/hare_mp/main.py

```diff
@@ -213,6 +213,10 @@ def reset(args):
         rc = 0
         util: ConsulUtil = ConsulUtil()
 
+        if is_cluster_running():
+            logging.info('Cluster is running, shutting down')
+            shutdown_cluster()
```

Comment:
def shutdown_cluster(skip_consul_stop: bool):
Do we need to check the default value for `skip_consul_stop`? and set to False?

Reply:
We always skip consul stop in mini provisioning